dev-util/coverity/files/coverity.envd

```diff
@@ -0,0 +1 @@
+PATH="/opt/coverity/bin"
```

Comment:
I think we usually prefer symlinks in /opt/bin over extra PATH entries. That is, if they work.

Reply:
The problem is that coverity expect to find locale, doc and such in the same root dir as the binary.

Reply:
Disregard this one then.